static/app/views/settings/organizationDeveloperSettings/sentryFunctionDetails.tsx

```diff
@@ -57,30 +75,47 @@ function SentryFunctionDetails(props: Props) {
 
   const handleSubmitSuccess = data => {
     addSuccessMessage(t('Sentry Function successfully saved.', data.name));
+    const baseUrl = `/settings/${orgId}/developer-settings/sentry-functions/`;
+    // TODO: should figure out where to redirect this
```

Comment:
Is this TODO still relevant?

Reply:
Yeah, I was considering bringing the user to a page displaying all sentry functions if they've saved their function for the first time, since that better mimics integration behavior.